src/components/Personalization/createExecuteDecisions.js

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Adobe. All rights reserved.
+Copyright 2020 Adobe. All rights reserved.
```

Comment:
If your editor is changing the year automatically, I don't think this is a problem, but if you're taking the time to manually change the year, you don't need to. Legal has said that the original year is fine and doesn't need to be updated as contents of the file change.

Reply:
I see, I didn't know that! thank you!